src/predict_average_embedding.py

```diff
@@ -0,0 +1,52 @@
+import os
+from pathlib import Path
+
+import torch
+import logging
+from torchtext.data.utils import ngrams_iterator
+from torchtext.data.utils import get_tokenizer
+import mlflow.pytorch
```

Comment:
IMPORTANT: this file will be saved to MLflow and then processed manually to import `ModelWrapper` from it, so this file **must not** have any dependency from `src`